
# Name of the task being performed
string taskName

# A string that represents the action type
string actionType

# Name of the current skill (within the task) being performed
string skillName

# Topics to record or playback.  Multiple topics can be listed, delimited by a space.
string topics

# True = playback, false = recording/demonstrating
bool playback

# Name of the run
string runName

# ID of the user
string userID
